### PR TITLE
chore: add blog introducing check-html-links

### PR DESCRIPTION
**Pull request by daKmoR**
## What I did

1.

### CODE REVIEW

File: docs/blog/introducing-check-html-links.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+| ---------------- | ------------- | -------- | ---- | --------------- |
+| check-html-links | 38            | ~2.5s    | node | 19              |
+| link-checker     | 3000+         | ~11s     | node | 106             |
+| hyperlink        | 68            | 4m 20s   | node | 481             |
+| htmltest         | 1000+         | ~0.7s    | GO   | -               |
+
+
+## Thanks
+
+Thank you for following along on my journey on creating `check-html-links`. You can find the code on [Github](https://github.com/modernweb-dev/rocket/tree/main/packages/check-html-links).
```
Comment (jlengrand) [inline docs/blog/introducing-check-html-links.md:160]: I'd probably add something about what you'd like to add to the tool or whether it is finished in your opinion

Comment (daKmoR) [reply docs/blog/introducing-check-html-links.md]: added a section above - thx for that tip 👍

File: docs/blog/introducing-check-html-links.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+Often these things happen as somewhere a page got moved or renamed and not every location got updated.
+It's really hard to catch especially if you have a dynamic page like with WordPress or an SPA.
+
+However, with the rise of SSG (Static Site Generators) this problem can be solved for good. The reason for that is that with all HTML rendered upfront as static files we can read all of them and check every link.
+
+## Evaluation
+
+Luckily I am not the first to think of such a solution so I tested a few tools.
+However, each had its own "blockers". Too slow, unclear output, too many dependencies, not node based, ...
```
[the remark this answers is not in the document]
Comment (daKmoR) [reply docs/blog/introducing-check-html-links.md]: it's now written differently (by you 😬) so I guess it's good now 🤗

File: docs/blog/introducing-check-html-links.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+It seems that most tools use sophisticated parsers meant to create full syntax trees of your html.
+However in reality for link checking all you need to know are the ids and the hrefs.
+
+I have been using [sax-wasm](https://github.com/justinwilaby/sax-wasm) in a few situations before and it seems to support streaming.
+So that could be fast 🤞
+
+So we know that theoretically, it should be possible to be fast - but what should be our goal?
+To not disturb too much during writing/development it should be finished within 1s for a small site (up to 200 pages).
+For medium sites (2000-1000 pages), it's reasonable if it takes a little longer - let's aim for less than 5 seconds. This will probably be a breaking point where you execute it only on-demand and in the CI instead of executing it on every save.
```
[the remark this answers is not in the document]
Comment (daKmoR) [reply docs/blog/introducing-check-html-links.md]: added 👍

**Pull request merged**